app/move/fields/create.js

```diff
@@ -261,12 +261,11 @@ module.exports = {
         text: 'fields::move_type.items.court_appearance.label',
         conditional: 'to_location_court_appearance',
       },
-      // TODO: Remove once we want to allow this journey
-      // {
-      //   value: 'prison',
-      //   text: 'fields::move_type.items.prison.label',
-      //   conditional: 'to_location_prison',
-      // },
+      {
```

Comment:
Are we ready to push this whole journey live already?

I'd be concerned that this will then make it available to any existing prison users. I think we still need to create the review workflow for this to be ready, right?

Reply:
That's correct. I am going to make a change to include an additional step as we discussed this morning.